sdk/monitor/azure-monitor-query/README.md

```diff
@@ -0,0 +1,316 @@
+- A metric name
+- The value itself
+- Some metrics may have multiple dimensions as described in Multi-dimensional metrics. Custom metrics can have up to 10
+  dimensions.
+
+## Examples
+
+* [Get logs for a query](#get-query "Get logs for a query")
+* [Get logs for a batch for queries](#get-batch-query "Get logs for a batch of queries")
+* [Get logs for a query with server timeout](#get-query-server-timeout "Get logs for a query with server timeout")
+* [Get metrics](#get-metrics "Get metrics")
+
+## Troubleshooting
+
+### Enable client logging
+
+You can set the `AZURE_LOG_LEVEL` environment variable to view logging statements made in the client library. For
+example, setting `AZURE_LOG_LEVEL=2` would show all informational, warning, and error log messages. The log levels can
+be found here: [log levels][log_levels].
+
+### Default HTTP Client
+
+All client libraries by default use the Netty HTTP client. Adding the above dependency will automatically configure the
+client library to use the Netty HTTP client. Configuring or changing the HTTP client is detailed in the
+[HTTP clients wiki](https://github.com/Azure/azure-sdk-for-java/wiki/HTTP-clients).
+
+### Default SSL library
+
+All client libraries, by default, use the Tomcat-native Boring SSL library to enable native-level performance for SSL
+operations. The Boring SSL library is an uber jar containing native libraries for Linux / macOS / Windows, and provides
+better performance compared to the default SSL implementation within the JDK. For more information, including how to
+reduce the dependency size, refer to the [performance tuning][performance_tuning] section of the wiki.
+
+## Next steps
+
+- Samples are explained in detail [here][samples_readme].
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to
+a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights
+to use your contribution.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate
+the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to
+do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see
+the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or
+comments.
+
+<!-- LINKS -->
+
+[azure_subscription]: https://azure.microsoft.com/free
+
+[jdk_link]: https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable
+
+[product_documentation]: https://aka.ms/awps/doc
+
+[log_levels]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java
+
+[performance_tuning]: https://github.com/Azure/azure-sdk-for-java/wiki/Performance-Tuning
```

Comment:
>https://github.com/Azure/azure-sdk-for-java/wiki/Performance-Tuning [](http://example.com/codeflow?start=22&length=67)

Where can I read more about this?

Reply:
You can find more details in the [Azure SDK Java OneNote](https://microsoft.sharepoint.com/:o:/r/teams/AzureDeveloperExperience/_layouts/15/WopiFrame.aspx?sourcedoc={f2251990-059c-48e6-8ff5-2c33680a30b0}&action=edit&wd=target%28Performance%2Eone%7C2F81917B%2D591C%2DAF4A%2D9DCE%2DB2E755E53F51%2F%29onenote%3Ahttps%3A%2F%2Fmicrosoft%2Esharepoint%2Ecom%2Fteams%2FAzureDeveloperExperience%2FShared%20Documents%2FLanguage%20%2D%20Java%2FAzure%20SDK%20%2D%20Language%20%2D%20Java%2FPerformance%2Eone#section-id={2F81917B-591C-AF4A-9DCE-B2E755E53F51}&end)